.github/workflows/ci.yaml

```
@@ -1,13 +1,19 @@
 name: CI
-on: push
+on:
+  - push
+  - pull_request
 jobs:
   test:
-    runs-on: ubuntu-20.04
     strategy:
+      fail-fast: false
       matrix:
+        os:
+          - ubuntu-20.04
+          - windows-2019
         node:
           - '18'
           - '16'
```

I suggest we try with the two last LTS versions:

```suggestion
          - '18'
          - '20'
```